sound/soc/sof/sof-priv.h

```diff
@@ -353,7 +353,9 @@ struct snd_sof_mailbox {
 struct snd_sof_ipc_msg {
 	/* message data */
 	u32 header;
+	u32 extension;
 	void *msg_data;
+	int msg_data_size;
```

Comment:
How is this relates to msg_size?

Reply:
> How is this relates to msg_size?

msg_data_size is not related to msg_size. It indicates the size of msg_data. We currently allocate msg_data to a fixed SOF_IPC_MSG_MAX_SIZE. We will allocate a different size for different platform.